### PR TITLE
Chores/update use of env vars #160877304

### PR DESCRIPTION
**Pull request by callaghanc**
After this PR is approved and merged, the team will:

1. Remove any DAHLIA-related env vars from their local environments.
2. Download the updated .env for LAP from LastPass.

### CODE REVIEW

File: .git-hooks/pre_commit/scan_for_env_vars.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+require 'dotenv/parser'
+
+module Overcommit
```
Comment (akegan) [inline .git-hooks/pre_commit/scan_for_env_vars.rb:3]: do you want help QAing this?

**Review by akegan — CHANGES_REQUESTED** [review]
Can you update the instructions for the e2e tests that say you need to have the env vars actually in your environment?

File: .git-hooks/pre_commit/scan_for_env_vars.rb (hunk 1 above)
Comment (callaghanc) [reply .git-hooks/pre_commit/scan_for_env_vars.rb:3]: would be great if any/all who are doing PR review can do QA as part of review - so, please actually try out everything in this PR locally and see if it works for you!

**Comment by callaghanc** [comment]
@akegan Updated e2e test section in README.

**Comment by akegan** [comment]
OK
- E2E tests worked for me with unset env vars
- Overcommit caught when I was using a value of an env var. The only headache it might cause is that I have yellow acres set as the default listing ID right now in https://github.com/Exygy/sf-dahlia-lap/blame/master/spec/javascript/support/puppeteer/consts.js#L2. We might run into trouble with those.

image

**Review by akegan — APPROVED** [review]

**Comment by callaghanc** [comment]
@akegan For the default listing/application ID in the consts file, it seems that if we have that as a default, a person would not need that same value in their local .env? Is that what you were referring to?

**Comment by akegan** [comment]
yep. I was thinking the same thing too but I just wanted to flag it.

**Comment by callaghanc** [comment]
yep makes sense. in general it will be something we'll have to consider if we in the future need to add an env variable with a common value - for example, if we need to add a flag env var whose value is just true/false. in that case, we'll need to add a way to exclude certain variables from this check. but we can handle that when that case arises.

**Pull request merged**